doc/source/randindex.rst

```diff
@@ -0,0 +1,33 @@
+Rand indices
+==========================
+
+`Rand index <http://en.wikipedia.org/wiki/Rand_index>`_ is a measure of the similarity between two data clusterings. From a mathematical standpoint, Rand index is related to the accuracy, but is applicable even when class labels are not used.
+
```

Comment:
Rand index applies to the case when the distribution of classes/memberships per assignment is different. So index relies on the assignment as in the case of ```varinfo```. Wiki article is a little bit confusing but the definition of Rand index is correct from my point of view.